openapi_tester/validators.py

```diff
@@ -114,13 +114,31 @@ def validate_type(schema_section: dict[str, Any], data: Any) -> str | None:
 
 
 def validate_format(schema_section: dict[str, Any], data: Any) -> str | None:
+    number_formats = ["number", "float", "double"]
```

Comment:
I'd suggest to make this list a constant (add in `constants.py`), so that it can be reused and don't need be re-defined here.

Reply:
Updated